src/jekyll/_layouts/showcase/spotlight.liquid

```diff
@@ -182,9 +184,17 @@
   {% if showcase.sections %}
     {% for section in showcase.sections %}
     <section class="wf-showcase__section-container">
-      <h3>{{section.title}}</h3>
-      {{ section.content || markdownify }}
+      <h3 id="{{ section.title | cgi_escape }}">{{section.title}}</h3>
```

Comment:
Totally good with this one, in an upcoming PR I'm going to change it to use `slugify` as it's a bit nicer looking.